principles.md

```diff
@@ -0,0 +1,34 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web, also known as web3. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+**Liberty**
+We believe in the sovereignty of individuals. As a platform that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms.
+
+**Censorship resistance**
+
+We aim for a free flow of information. No content is under surveillance. One of the design goals of Ethereum is to be censorship resistant, and we should provide the same guarantees across Status.
+
+**Security**
+We don't compromise on security when building features. We will use state-of-the-art technologies, and research new security methods and technologies to make strong security guarantees.
+
+**Privacy**
+Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as to preserve the right to total anonymity.
+
+**Transparency**
+We strive for complete openness and symmetry of information, and have no border between our core contributors and our community. We will be frank about our shortcomings, especially when making short-term tradeoffs in service of our long-term goals.
+
+**Free culture**
+The software we create is a public good. It will be made available via a free and open source license, for anyone to share, modify and benefit from.
+
+**Decentralization**
+We will minimize centralization across both the software and the organization itself. In other words, we will maximize the number of physical computers composing the network, and maximize the number of individuals who have control over the system(s) we are building.
+
+**Inclusivity**
+We believe in fair and widespread access to our software, with an emphasis on ease-of-use. This also extends to social inclusivity, permissionless participation, and investing in educational efforts.
+
+**Continuance**
+We aim to create software that will continue to exist and improve, without the stewardship of a single entity or any of the current team members.
+
+**Resourcefulness**
+Doing more with less. By optimizing what we have to work with, we aim to solve problems in the most effective way possible at lower economic costs (in terms of capital, time and resources).
```

Comment:
Maybe add that this resourcefulness is important for the `sustainability` of Status (to avoid perceptions as mentioned above by @yenda that this is about being cheap)?